checkov/main.py

```diff
@@ -683,7 +683,7 @@ def run(self, banner: str = checkov_banner) -> int | None:
                 logger.debug('Using --list; setting source to DISABLED')
                 source = SourceTypes[BCSourceType.DISABLED]
 
-            if CHECKOV_RUN_SCA_PACKAGE_SCAN_V2 and source.upload_results:
+            if CHECKOV_RUN_SCA_PACKAGE_SCAN_V2:
                 self.runners.append(sca_package_runner_2())
             else:
                 self.runners.append(sca_package_runner())
```

Comment:
also here I think we dont need this anymore, and need to add the runner 2 to runner list `DEFAULT_RUNNERS`